src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -7,6 +7,7 @@ import ChannelAnonymousImage from './ChannelAnonymousImage';
 import ChannelAnonymousSubImage from './ChannelAnonymousSubImage';
 import ChatIcon from '../../../assets/chat-icon.png';
 import FeedIcon from '../../../assets/images/feed-icon.png';
+import dimen from '../../../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `dimen` is a new addition to this file. Ensure that the `dimen` utility is properly implemented and tested, as it's used throughout the code for normalizing dimensions.
```diff
+ import dimen from '../../../utils/dimen';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->